neuro-cli/src/neuro_cli/utils.py

```diff
@@ -616,7 +616,7 @@ def parse_permission_action(action: str) -> Action:
 def format_size(value: Optional[float]) -> str:
     if value is None:
         return ""
-    return humanize.naturalsize(value, gnu=True)
+    return humanize.naturalsize(value)
```

Comment:
This change is responsible for many test changes, and is not essential for functionality. I would make it in a separate PR.

Reply:
Good point, I've reverted this change.